### PR TITLE
IDEX-3852: command selector improvements

### PR DESCRIPTION
**Pull request by olexii4**
@ashumilova WDYT?

### CODE REVIEW

File: ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/action/Separator.java [hunk 1]
```diff
@@ -22,6 +21,7 @@
     private String myText;
 
     public Separator() {
+        this(null);
```
Comment (sleshchenko) [inline ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/action/Separator.java]: Can you explain why do we need this line?

Comment (vzhukovskii) [inline ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/action/Separator.java]: I gues to initialize separator with null text

Comment (sleshchenko) [inline ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/action/Separator.java]: I thought It is null by default :) But Oleksii told me that there is some problem with undefined and null value. Thanks for explanation.

**Comment by vzhukovskii** [comment]
+1

**Pull request merged**